### PR TITLE
[FIX] Message sequential after message thread preview

### PR DESCRIPTION
**Pull request by filipemarins**
## Further comments

Message sequential after message thread preview

Before:
Screen Shot 2022-09-19 at 11 37 36

After:
Screen Shot 2022-09-19 at 11 37 09

### CODE REVIEW

**Review by guijun13 — DISMISSED** [review]
LGTM

File: apps/meteor/client/views/room/MessageList/lib/isMessageSequential.spec.ts [hunk 1]
```diff
@@ -135,4 +135,16 @@ describe('isMessageSequential', () => {
...
+			...baseMessage,
+			tmid: 'threadId',
+			tshow: true,
+		};
+		const current: IMessage = {
+			...baseMessage,
+		};
+		expect(isMessageSequential(current, previous, TIME_RANGE_IN_SECONDS)).to.be.false;
+	});
 });
```
Comment (hugocostadev) [inline apps/meteor/client/views/room/MessageList/lib/isMessageSequential.spec.ts:150]: Hi @filipemarins , if you check the file `.mocha.client.js` you will notice that just test files that are inside `tests/unit/**/*` are included in the pipeline, so you need to change to the expected path.
Screen Shot 2022-09-20 at 8 18 40 AM

I already did in [this PR](https://github.com/RocketChat/Rocket.Chat/pull/26751) but it's not merged yet.

**Review by hugocostadev — CHANGES_REQUESTED** [review]

**Review by hugocostadev — DISMISSED** [review]

File: apps/meteor/client/views/room/MessageList/lib/isMessageSequential.ts [hunk 2]
```diff
@@ -28,5 +28,9 @@ export const isMessageSequential = (current: IMessage, previous: IMessage | unde
 		return false;
 	}
 
+	if (isThreadMessage(previous) && previous.tshow) {
```
Comment (ggazzo) [inline apps/meteor/client/views/room/MessageList/lib/isMessageSequential.ts:31]: what about create something like 'isThreadAnswer'

**Review by ggazzo — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/room/MessageList/lib/isMessageSequential.ts [hunk 3]
```diff
@@ -28,5 +28,9 @@ export const isMessageSequential = (current: IMessage, previous: IMessage | unde
...
+		return false;
+	}
```
Comment (ggazzo) [inline apps/meteor/client/views/room/MessageList/lib/isMessageSequential.ts:33]: please break this test into another function, in that way its not possible to use this function inside threads for example

**Review by ggazzo — CHANGES_REQUESTED** [review]

**Review by ggazzo — APPROVED** [review]

**Pull request merged**